types/moment-timezone/moment-timezone.d.ts

```diff
@@ -42,9 +47,13 @@ declare module 'moment' {
     load(data: { version: string; links: string[]; zones: string[] }): void;
 
     names(): string[];
+    zonesForCountry<T extends true>(country: string, with_offset: T): T extends true ? MomentZoneOffset[] : never;
+    zonesForCountry<T extends false>(country: string, with_offset?: T): T extends false ? string[] : never;
+    zonesForCountry(country: string, with_offset?: boolean): MomentZoneOffset[] | string[];
+    countries(): string[];
```

Comment:
The zonesForCounty() method has 2 overloads.  The 2 overloads determine the return type based on the value of the with_offset parameter.

Reply:
Additionally the counties() method is added.  In another pull request for adding zoneForCounties(), this method was omitted.

Reply:
I think the second parameter is an object.
```
moment.tz.zonesForCountry(String, { offset: true });
```
[source](https://momentjs.com/timezone/docs/#/using-timezones/getting-country-zones/)

Reply:
@justindra It sure is!  I will fix this now.  Good catch ;)

Reply:
@justindra Actually it looks like the documentation is wrong on the moment-timezone documentation page, because according to the source, the parameter is a boolean.  See: https://github.com/moment/moment-timezone/blob/0.5.28/moment-timezone.js#L503